test/models/confirmable_test.rb

```diff
@@ -225,6 +225,11 @@ def setup
     user = create_user
     user.confirmation_sent_at = Time.zone.today
     refute user.active_for_authentication?
+
+    Timecop.freeze(Time.zone.today) do
```

Comment:
Can you create a new test block for these new assertions? It would be easier to read.